rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -720,48 +721,6 @@ pub mod cycles_ledger {
     }
 }
 
-/// Installs the NNS canisters, ensuring that there is a whale neuron with `TEST_NEURON_1_ID`.
-/// Requires PocketIC to have at least an NNS and an SNS subnet.
-///
-/// Arguments
-/// 1. `with_mainnet_nns_canister_versions` is a flag indicating whether the mainnet
-///    (or, therwise, tip-of-this-branch) WASM versions should be installed.
-/// 2. `with_test_nns_governance_canister` is a flag indicating whether the test version of
-///    the governance canister should be installed. Mutually exclusive with `with_mainnet_nns_canister_versions`.
-/// 3. `initial_balances` is a `Vec` of `(test_user_icp_ledger_account,
-///    test_user_icp_ledger_initial_balance)` pairs, representing some initial ICP balances.
-/// 3. `custom_registry_mutations` are custom mutations for the inital Registry. These
-///    mutations should comply with Registry invariants, otherwise this function will fail.
-/// 4. `neurons_fund_hotkeys` - hotkeys of the 1st NNS (Neurons' Fund-participating) neuron.
-///
-/// Returns
-/// 1. A list of `controller_principal_id`s of pre-configured NNS neurons.
-pub async fn install_nns_canisters(
```

Comment:
Note for reviewer - this was inlined b/c the builder is better and adding an additional parameter to use the test version of governance seemed like the wrong direction to go in.